schematron/Consolidation.sch

```diff
@@ -2810,8 +2810,6 @@ Schematron generated from Trifolia on 7/20/2021
       <sch:assert id="a-1198-8903" test="count(cda:code)=1">SHALL contain exactly one [1..1] code, which SHOULD be selected from ValueSet Health Insurance Type urn:oid:2.16.840.1.113883.3.88.12.3221.5.2 DYNAMIC (CONF:1198-8903).</sch:assert>
       <sch:assert id="a-1198-19109" test="cda:statusCode[@code='completed']">This statusCode SHALL contain exactly one [1..1] @code="completed" Completed (CodeSystem: HL7ActStatus urn:oid:2.16.840.1.113883.5.14 STATIC) (CONF:1198-19109).</sch:assert>
       <sch:assert id="a-1198-15992-c" test="not(cda:performer/cda:assignedEntity[cda:templateId[@root='2.16.840.1.113883.10.20.22.4.87']]/cda:code) or cda:performer/cda:assignedEntity[cda:templateId[@root='2.16.840.1.113883.10.20.22.4.87']]/cda:code[@code]">The code, if present, SHALL contain exactly one [1..1] @code, which SHOULD be selected from ValueSet Financially Responsible Party Type Value Set urn:oid:2.16.840.1.113883.1.11.10416 DYNAMIC (CONF:1198-15992).</sch:assert>
-      <sch:assert id="a-1198-10484-c" test="cda:participant[@typeCode='COV']/cda:participantRole/cda:addr[cda:streetAddressLine and cda:city and (cda:country!='US' or ((not(cda:country) or cda:country!='US') and cda:state and cda:postalCode))]">The content of addr **SHALL** be a conformant US Realm Address (AD.US.FIELDED) (2.16.840.1.113883.10.20.22.5.2) (CONF:1198-10484).</sch:assert>
```

Comment:
At a glance, I'm not seeing the templateIds for these two conformance statements present in the context list for the US Realm Address rule, although maybe I'm just missing something here.

Reply:
I see ` | cda:act[cda:templateId[@root='2.16.840.1.113883.10.20.22.4.61' and @extension='2015-08-01']]/cda:participant/cda:participantRole/cda:addr` at the bottom of line 520. And if you try to break the US-Realm-Address rules in a Policy Activity's participant, it does fire.

It does seem odd that Trifolia's not pulling in the `[@typeCode...]` branch identifier; instead it seems to just be requiring that ALL participants in this template match US realm address. To be fair, it looks like it's doing that for performers, too (and likely plenty of other places). While this seems a bit over-specified, it doesn't seem that egregious to require a US Realm address in additional places inside a US Realm document.  (it's not like it _requires_ the country to be US)

Reply:
Ah... I'd been looking for the `2.16.840.1.113883.10.20.22.4.89` templateId. Weird that the Policy Activity works this way in the first place (where we've got those required templateIds but they don't correspond to named templates in Trifolia) but I guess given that we do, this is the way we'd expect the schematron to work.